text/0000-intro-video-script.md

```diff
@@ -0,0 +1,73 @@
+# Meta
+[meta]: #meta
+- Name: Intro video script
+- Start Date: 2021-05-18
+- Author(s): yaelharel
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+Improving documentation is a project priority for 2021.
+This RFC is an attempt to create an easy and clear starting point for new users and contributors.
+
+# Definitions
+[definitions]: #definitions
+
+N/A
+
+# Motivation
+[motivation]: #motivation
+
+It is currently relatively challenging for new users and contributors to understand what buildpacks are and what is the purpose of our project.
+It can be very useful to have a ~1 minute introductory video to provide high-level answers to these questions.
+We can post this video on our website and share it on social media.
+Hopefully this will help familiarize more people with our project.
+
+
+# What it is
+[what-it-is]: #what-it-is
+
+### The script
+
+When was the last time you struggled to wrangle yet another Dockerfile? As an application developer, you should only have to focus on what you do best - writing code, without having to worry about your container build strategy. Buildpacks can help you with just that! They are a better approach to building container images for applications.
+
+Buildpacks are programs that transform source code into container images by analyzing the code and determining the best way to build it. They can provide dependencies, configuration, run compilation, and much more! A buildpack can be written by anyone, and shared with everyone! Google, Heroku/Salesforce, and the Paketo project provide buildpacks for a wide variety of language families, and you can discover even more by searching the buildpack registry. With buildpacks, you can use best-in-class container build strategies from industry experts with ease.
+
+Even more so, the Cloud Native Buildpacks project provides a specification for buildpack authors and platforms that ensures builds are fast, secure, customizable, and can run on any cloud. It is a Cloud Native Computing Foundation project with many existing integrations and many more to come.
+
+You are invited to join our community. Try buildpacks with your favorite language today.
+
+
+# How it Works
+[how-it-works]: #how-it-works
+
+After this RFC will get accepted and merged, we will start working on the actual video. 
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+N/A
```

Comment:
I think there's a drawback that maintaining the video is harder than editing text.

Reply:
I'll add it, although I think that we won't need to change it anytime soon because it's very basic.

Reply:
Please see https://github.com/buildpacks/rfcs/pull/161/commits/084534fe1e7cf27c4589ea6215633518a0a5b9d6.